textile/chat-features.textile

```diff
@@ -344,10 +350,11 @@ Presence allows chat room users to indicate to others that they're online, as we
 ** @(CHA-PR6b)@ This specification point has been removed.
 *** @(CHA-PR6b1)@ This specification point has been removed.
 *** @(CHA-PR6b2)@ This specification point has been removed.
-** @(CHA-PR6c)@ @[Testable]@ If the room status is @Attaching@, then the @get@ call will wait for the attach operation to succeed and then call the underlying presence @get@ operation. It will throw an error if the attach fails.
-** @(CHA-PR6d)@ @[Testable]@ If the room status is @Attached@, then the @get@ call will invoke the underlying @presence.enter()@ call.
-** @(CHA-PR6e)@ @[Testable]@ If the room status is @Detached@, an @ErrorInfo@ with code 40000, explaining that attach must be called first, will be thrown.
-** @(CHA-PR6f)@ @[Testable]@ For any other room status, an @ErrorInfo@ with code 40000 will be thrown.
+** @(CHA-PR6c)@ @[Testable]@ If the room status is @Attaching@, then the operation shall enter a @CHA-RL9@ pending room status operation. If this succeeds (i.e the room becomes @ATTACHED@), the library must invoke the underlying @presence.get()@ call.
```

Comment:
```suggestion
** @(CHA-PR6c)@ @[Testable]@ If the room status is @Attaching@, then the operation should wait as per @CHA-RL9a@, once RoomStatus is @Attached@ as per @CHA-RL9b@ then the library must invoke the underlying @presence.get()@ call, otherwise throw error as per @CHA-RL9c@
```

Reply:
Will mark this as resolved and continue discussion on the one I've left open